libs/community/langchain_community/vectorstores/chroma.py

```diff
@@ -610,11 +611,15 @@ def get(
 
         return self._collection.get(**kwargs)
 
+    @deprecated(since="0.1.17", removal="0.3.0")
```

Comment:
could we add a `message="..."` specifying that manual persistence is no longer supported bc as of chroma 0.4 docs are automatically persisted

Reply:
Thanks for the feedback!